test/e2e/node_feature_discovery_test.go

```diff
@@ -215,6 +215,18 @@ var _ = SIGDescribe("NFD master and worker", func() {
 					testpod.SpecWithContainerImage(dockerImage()),
 					testpod.SpecWithTolerations(testTolerations),
 					testpod.SpecWithContainerExtraArgs("-enable-taints"),
+					testpod.SpecWithContainerExtraArgs(
+						fmt.Sprintf(
+							"-deny-label-ns=%v",
+							strings.Join([]string{"*.denied.ns", "random.unwanted.ns", "*.vendor.io"}[:], ","),
+						),
+					),
+					testpod.SpecWithContainerExtraArgs(
+						fmt.Sprintf(
+							"-extra-label-ns=%v",
+							strings.Join([]string{"custom.vendor.io"}[:], ","),
+						),
+					),
```

Comment:
Is there a reason you want to use Sprintf instead of simple one-liner

```suggestion
					testpod.SpecWithContainerExtraArgs("-extra-label-ns=custom.vendor.io"),
```

Also applies to the `-deny-label-ns` above...